pages/vi/firststeps.md

```diff
@@ -145,6 +147,8 @@ Make sure you have created at least one issue, resolved it, commented on an issu
 * 1 comment added (on an issue you didn't create)
 * 1 issue created
 
+Check your progress [here](https://treehouses.io/#!./pages/vi/trackprogress.md)
```

Comment:
please do not call the whole url if shorter options like relative ones or just filenames are working

Reply:
Sorry about that. I fixed it.